integration-tests/simple-webapp-integration-test/src/test/java/co/elastic/apm/servlet/AbstractServletContainerIntegrationTest.java

```diff
@@ -119,17 +119,13 @@ protected AbstractServletContainerIntegrationTest(GenericContainer<?> servletCon
         this.servletContainer = servletContainer;
         this.webPort = webPort;
         this.contextPath = contextPath;
-        // uncomment for debugging
-        //this.debugProxy = createDebugProxy(servletContainer, 5005);
+        this.debugProxy = createDebugProxy(servletContainer, 5005);
```

Comment:
please make sure the debug proxy is not started by default. It makes the tests fail in the new pipeline, maybe there is a different level of testing concurrently there. I will make all of it easier for us using a single change to enable debugging.